sources/packages/backend/apps/api/src/route-controllers/designation-agreement/designation-agreement.institutions.controller.ts

```diff
@@ -72,7 +72,8 @@ export class DesignationAgreementInstitutionsController extends BaseController {
       "Not able to create a designation agreement due to an invalid request.",
   })
   @ApiUnprocessableEntityResponse({
-    description: "Your institution already has one pending designation request; you cannot submit another one until the first has been approved or denied.",
+    description:
```

Comment:
Is this intended to be part of PR?

Reply:
No but i guess the prettier just added it, i can remove if its not needed.

Reply:
I would recommend to remove to avoid confusion.